recipes/geohash2/LICENSE.txt

```diff
@@ -0,0 +1,661 @@
+                    GNU AFFERO GENERAL PUBLIC LICENSE
```

Comment:
It looks like the license file is already distributed with the source code, so I think you can remove this recipe-local copy.

Reply:
Ok. Going to delete the license.txt

Reply:
Deleted the license.txt. Please let me know if everything is ok